sql/database.sql

```diff
@@ -5384,6 +5384,8 @@ CREATE TABLE `pharmacies` (
   `name` varchar(255) default NULL,
   `transmit_method` int(11) NOT NULL default '1',
   `email` varchar(255) default NULL,
+  `ncpdp` int(12) DEFAULT NULL,
+  `npi` int(12) DEFAULT NULL.
   PRIMARY KEY  (`id`)
 ) ENGINE=InnoDB;
```

Comment:
I don't understand. I looked in the main repository and those columns are not there for me to copy.